client/blocks/free-to-paid-banner/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Banner from 'components/banner';
+import {
+	PLAN_PERSONAL,
+} from 'lib/plans/constants';
+import { abtest } from 'lib/abtest';
+import {
+	eligibleForFreeToPaidUpsell,
+} from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const FreeToPaidBanner = ( props ) => {
+	if ( ! props.eligibleForFreeToPaidUpsell || abtest( 'freeToPaidUpsell' ) !== 'banner' ) {
+		return null;
+	}
+
+	const { translate = noop } = props;
+	const title = translate( 'Get a free custom domain name with a WordPress.com plan!' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Get a free custom domain name (example.com) with this plan to use for your website.' )` *ES Score: 7.10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/free-to-paid-upsell)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).